web/src/composebox_typeahead.ts

```diff
@@ -43,6 +43,12 @@ import type {UserPillData} from "./user_pill.ts";
 import {user_settings} from "./user_settings.ts";
 import * as util from "./util.ts";
 
+/* Maximum length for stream name with markdown is 66.
+   If the look back is less than this, topic completion
+   doesn't work for long stream names.
+*/
+const MAX_LOOK_BACK_FOR_TYPEAHEAD_COMPLETION = 66;
```

Comment:
I think we might want this to be 60 + 4 + 20, to give 20ish characters for a topic?